ols/src/query_helpers/docs_summarizer.py

```diff
@@ -49,6 +52,7 @@ def summarize(self, conversation: str, query: str, **kwargs) -> tuple[str, str]:
         )
         logger.info(f"{conversation} call settings: {settings_string}")
 
+        # TODO: use history there
```

Comment:
Where it is passed in? Or that is not implemented yet?

Reply:
in ols query handler

Reply:
I mean here, I don't see arg `history` used in this `summarize` method.

Reply:
I don't understand the benefit of having the work splitter then.
But there is no reason to block this PR though.

Reply:
i'm also not clear on the value of this as a standalone PR but if it's how you prefer to land the work i guess it's ok.